ProductMicro/src/main/java/com/grizzly/productmicro/ProductController.java

```diff
@@ -34,7 +34,7 @@ public ResponseEntity get(@PathVariable(value="pageIndex") Integer pageIndex, @P
             return buildResponse(new ApiError(HttpStatus.NOT_FOUND, "No products were found.", "pageIndex: " + pageIndex + "; column_name: " + column_name));
         }
 
-        return new ResponseEntity(products, HttpStatus.OK);
+        return new ResponseEntity<>(products, HttpStatus.OK);
```

Comment:
Just FYI (not an issue, but yeah) you don't actually need these container brackets here.

Reply:
I put the containers because IntelliJ was complaining about it ...